build.gradle

```diff
@@ -21,5 +21,5 @@ repositories {
     maven { url 'https://repo.leanplum.com' }
 }
 dependencies {
-    api 'com.leanplum:Leanplum:3.0.1'
+    api 'com.leanplum:Leanplum:3+'
```

Comment:
Unfortunately we can't use wildcards in our dependencies. With wildcards, you never actually know what version of a dependency you're building with. Your QA app for example could build with one version, and then when you build the app for production it could suddenly be a different version. If there's an internet connectivity issue on the machine, it'll use whatever version it has in its various gradle caches, making things even less deterministic for our customers. 

iOS doesn't have the same issue, where developers are conditioned to use a Podfile.lock file to ensure they always have the same version in place. that's why we can use wildcards there but not w/ gradle.